syntax = "proto3";

package cloud;

message BinlogCloudItem {
  //belong to which db
  uint64 db_id      = 1;
  //belong to whicn rocksdb
  uint64 rocksdb_id = 2;
  //data write time
  uint64 exec_time  = 3;
  uint64 term_id    = 4;
  uint64 file_num   = 5;
  uint64 offset     = 6;
  bytes content     = 7;
  uint64 type       = 8;
}
